.all-contributorsrc

```diff
@@ -39,7 +39,9 @@
       "avatar_url": "https://avatars.githubusercontent.com/u/4550621?v=4",
       "profile": "https://github.com/Twixes",
       "contributions": [
-        "code"
+        "code",
+        "test",
+        "design"
```

Comment:
What about `"ideas"`? No ideas? What about `"bug"`? I've introduced many bugs. Also fixed some. Written some `"doc"`s too. Did some `"userTesting"`. Give credit where it's due!11